.github/dependabot.yaml

```diff
@@ -74,3 +74,21 @@ updates:
     open-pull-requests-limit: 1
     reviewers:
     - "stackrox/backend-dep-updaters"
+
+  - package-ecosystem: 'docker'
+    directory: 'image/rhel'
+    schedule:
```

Comment:
1. Could you create a reusable fragment with `schedule`, `open-pull-requests-limit` and `reviewers` instead of copy-pasting it please? I know that GitHub may be restricting YAML anchors, but worth a try.
2. Would be nice if you could add `operator` and `docs` directories.

Reply:
https://github.com/dependabot/dependabot-core/issues/1582

Reply:
Looks like it's not supported

Reply:
Operator is already there and docs uses latest so there is no need to dependabot

Reply:
My bad. Did not check operator above.